jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "rootDir": "./",
+    "baseUrl": "./"
+  }
+}
```

Comment:
If using VSCode, this allows us to `ctrl+click` directly to anything written in JS, like we would in TypeScript. Unfortunately it doesn't work with `.coffee` files tho :(